packages/dashboard/e2e/env.sh

```diff
@@ -5,3 +5,9 @@ export REACT_APP_TRAJECTORY_SERVER=${REACT_APP_TRAJECTORY_SERVER:-ws://localhost
 export REACT_APP_API_SERVER=${REACT_APP_API_SERVER:-ws://localhost:50002}
 export ROMI_DASHBOARD_PORT=${ROMI_DASHBOARD_PORT:-5000}
 export COMPOSE_PROJECT_NAME=${COMPOSE_PROJECT_NAME:-romidashboarde2e}
+
+if [ -z "${CI}" ]; then
```

Comment:
One possible use of `CI` locally is to run the tests in headless mode.

If using proposal 1, we should be able to use the same logic on both gh and local runs. If using proposal 2, I think it's better to use `GITHUB_NETWORK` to check the environment.

Reply:
Are we going to run the tests in headless mode locally?